### PR TITLE
Is loading

### PR DESCRIPTION
**Pull request by biharck**
### What
Fixing isLoading issue.

### CODE REVIEW

File: src/CornerstoneViewport/CornerstoneViewport.js [hunk 1]
```diff
@@ -445,7 +441,8 @@ class CornerstoneViewport extends Component {
 
         // Display the first image
         cornerstone.displayImage(element, image, viewport);
-
+        const endLoadHandler = cornerstoneTools.loadHandlerManager.getEndLoadHandler();
+        endLoadHandler(element);
```
Comment (swederik) [inline src/CornerstoneViewport/CornerstoneViewport.js]: We should guard this. `endLoadingHandler` might be undefined

File: src/CornerstoneViewport/CornerstoneViewport.js [hunk 2]
```diff
@@ -803,23 +809,6 @@ class CornerstoneViewport extends Component {
     });
   };
 
-  startLoadingHandler = () => {
```
Comment (swederik) [inline src/CornerstoneViewport/CornerstoneViewport.js]: I'm wondering if we should check if `cornerstoneTools.loadHandlerManager.getEndLoadHandler()` and `cornerstoneTools.loadHandlerManager.getStartLoadHandler()` exist when we are doing componentDidMount. If they don't exist, we could set them to do this setState behaviour. That way if the end user doesn't set them up and just uses one CornerstoneViewport component somewhere, the loading functionality would just work out of the box. Thoughts?

If they used two of these components though, they would need to move the handler logic to a higher level. Might be confusing to end users.

File: src/CornerstoneViewport/CornerstoneViewport.js (hunk 1 above)
Comment (biharck) [reply src/CornerstoneViewport/CornerstoneViewport.js]: Done

File: src/CornerstoneViewport/CornerstoneViewport.js (hunk 2 above)
Comment (biharck) [reply src/CornerstoneViewport/CornerstoneViewport.js]: Actually, it is already done at https://github.com/OHIF/react-viewerbase/blob/a94e3a9ff970185d220aa7381e48d4d82de004a9/src/LayoutChooser/LayoutManager.js#L63-L65

**Comment by biharck** [comment]
I think we can close this branch @swederik

**Pull request closed without merging**